multimodal/src/autogluon/multimodal/configs/environment/default.yaml

```diff
@@ -1,5 +1,5 @@
 env:
-  num_gpus: -1
+  num_gpus: 0
```

Comment:
No need to modify the default here. We can add
`env.num_gpus: 0` in the detection's preset.

Reply:
Done